apps/login-register/src/app/login/login.component.ts

```diff
@@ -41,5 +59,26 @@ export class LoginComponent implements OnInit {
       email: ['', [Validators.required, Validators.email]],
       password: ['', [Validators.required, Validators.minLength(8)]],
     });
+
```

Comment:
extract this into a separate method and use it in the onInit

Reply:
it is already in a private method which is called in the onInit cycle

![image](https://user-images.githubusercontent.com/130452422/236382816-051392d1-e695-4675-9e5e-4b1ded2bf791.png)

![image](https://user-images.githubusercontent.com/130452422/236382858-9726a56f-d32b-4fc6-b6b8-37cb6f196b6e.png)